pkg/controller/controller.go

```diff
@@ -61,43 +81,91 @@ func NewController(client kubernetes.Interface, pvcInformer cache.SharedIndexInf
 }
 
 func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
-	defer c.queue.ShutDown()
+	defer c.pvcQueue.ShutDown()
```

Comment:
don't need to also shutdown the podQueue?

Reply:
yep! will fix, thanks.